lib/french_toast.rb

```diff
@@ -1,3 +1,4 @@
+require "redis"
```

Comment:
Should we require this in `notifier.rb` instead, to keep everything together?

Reply:
Good call! Updated.